wandb/sdk/interface/artifacts/artifact.py

```diff
@@ -23,7 +23,7 @@ def __init__(
         super().__init__(msg.format(artifact=artifact, attr=attr, method_id=method_id))
         # Follow the same pattern as AttributeError.
         self.obj = artifact
-        self.name = attr
+        self.name = attr or ""
```

Comment:
Not sure why you changed this.

Reply:
Mypy complained; I'm not actually sure why mypy isn't complaining in `main` though.